tokio-quiche/src/quic/io/worker.rs

```diff
@@ -355,11 +363,15 @@ where
             }
 
             #[cfg(not(feature = "gcongestion"))]
-            let max_send_size = tune_max_send_size(
-                segment_size,
-                qconn.send_quantum(),
-                send_buf.len(),
-            );
+            let max_send_size = if !use_get_next_release_time {
+                tune_max_send_size(
+                    segment_size,
+                    qconn.send_quantum(),
+                    send_buf.len(),
+                )
+            } else {
+                usize::MAX
```

Comment:
can you add a comment that this is "disabling" quantum by setting it to max

Reply:
done.